lib/ui/src/core/store.js

```diff
@@ -1,33 +1,50 @@
-// simple store implementation
-const createStore = initialState => {
-  let state = initialState;
-  const listeners = [];
+// TODO -- make this TS?
 
-  function notify() {
-    listeners.forEach(l => l());
+import { localStorage, sessionStorage } from 'global';
```

Comment:
We might be better off using indexedDB, since we'd not have to stringify and parse.

Reply:
I don't really know if it's a big deal. Easy to change after the fact as this is hidden behind an API.